test/wdio.conf.js

```diff
@@ -31,6 +31,89 @@ async function capabilityDefaults() {
   };
 }
 
+class GitHubStatus {
+  /**
+   * @param {Object} p
+   * @param {string} p.sha
+   * @param {string} p.repo
+   * @param {string} p.token
+   * @param {string} p.context
+   */
+  constructor({sha, repo, token, context}) {
+    this.sha = sha;
+    this.repo = repo;
+    this.token = token;
+    this.context = context;
+  }
+
+  setTarget(priority, target) {
+    if (this.targetPriority > priority) {
```

Comment:
It seems that `undefined > t` is false for any `t`, but... 🤮

Reply:
Obligatory https://www.destroyallsoftware.com/talks/wat